Makefile

```diff
@@ -16,6 +16,7 @@ DEPEND=\
 	github.com/goadesign/goa.design/tools/godoc2md \
 	github.com/golang/lint/golint \
 	github.com/onsi/ginkgo \
+	github.com/spf13/hugo \
```

Comment:
That's not needed here correct? (`make docs` already gets it - one less dependency is nice :) )